src/run.ts

```diff
@@ -37,6 +38,24 @@ export async function run() {
       const publicFqdnInput = core.getInput('public-fqdn') || ''
       const publicFqdn = publicFqdnInput.toLowerCase() === 'true'
 
+      // check resource type is recognized
+      if (
+         resourceType.toLowerCase() !== 'microsoft.containerservice/fleets' &&
+         resourceType.toLowerCase() !== 'microsoft.containerservice/managedclusters'
+      ) {
+         throw Error(
+            'Resource type not recognized, either Microsoft.ContainerService/managedClusters or Microsoft.ContainerService/fleets is valid'
+         )
+      }
+
+      //validate user is not using admin or publicFqdn flags with fleet resource
+      if (resourceType === 'microsoft.containerservice/fleets' && admin) {
```

Comment:
use toLowerCase() here and below

Reply:
You'd probably be easiest writing something like `const resourceType = (core.getInput('resource-type') || 'Microsoft.ContainerService/managedClusters').toLowerCase()` and remove it from everywhere else?

Reply:
(and do the same in the k8s-deploy PR)